resources/public/api/conf/1.0/errors.raml

```diff
@@ -159,6 +192,17 @@ traits:
                 description: The supplied accounting period does not exist
                 value:
                   code: RULE_NO_ACCOUNTING_PERIOD
+  ruleNoAdjustmentsMade:
+    responses:
+      400:
+        body:
+          application/json:
+            type: types.errorResponse
+            examples:
+              ruleNoAdjustmentsMade:
+                description: An Adjusted calculation doesn't exist
```

Comment:
`An Adjusted calculation` -> `An adjusted calculation` ?

Reply:
Taken from the requirements spec as it appeared.